### DISCUSSION TITLE
Copyright Salesforce?

### DISCUSSION THREAD

**Discussion by kvnlnt** [alpinejs/alpine#655] [4. General]
Why is there a "Copyright (C) 2017 salesforce.com, inc." around line 900 in dist/alpine.js? Doesn't appear to be in the src but it's in the dist?

**Comment by ryangjchandler** [comment]
@kvnlnt Alpine uses a JavaScript library developed by Salesforce called `mutation-observer`. This handles all of the reactivity in the core.

It won't be in the source since this module is only bundled at build.
